src/subsys/ngx_subsys_lua_output.c.tt2

```diff
@@ -698,6 +706,10 @@ ngx_[% subsys %]_lua_ngx_eof(lua_State *L)
                                | NGX_HTTP_LUA_CONTEXT_CONTENT);
 
```

Comment:
Let's remove blank line here otherwise we would have an extra blank line in the C output.